app/controllers/v0/profile/transactions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class TransactionsController < ApplicationController
+
+      # before_action { authorize :vet360, :access? }
+      
+      def status
+        transaction = AsyncTransaction::Vet360::Base.refresh_transaction_status(@current_user, transaction_params[:transaction_id])
+        raise Common::Exceptions::RecordNotFound, transaction unless transaction
+        render json: transaction
```

Comment:
You'll want to specify the serializer

Reply:
Thanks. I thought those got matched automagically by the names... I'm never quite sure where Rails magic starts and ends ;-)